lib/response.rb

```diff
@@ -1,24 +1,38 @@
 class Response
-  attr_reader :verb, :path, :protocol, :host, :port, :origin, :accept
+  attr_accessor :status_code
+  attr_reader :verb, :path, :protocol, :host, :port, :origin,
+              :accept, :guess
 
   def initialize
-    @verb     = ""
-    @path     = ""
-    @protocol = ""
-    @host     = ""
-    @port     = ""
-    @origin   = ""
-    @accept   = ""
+    @status_code = "200 ok"
+    @guess       = nil
   end
 
   def process_lines(request_lines)
     @verb     = request_lines[1][0]
     @path     = request_lines[1][1]
     @protocol = request_lines[1][2]
-    @host     = request_lines.last[1].split(':')[0]
+    @host     = request_lines[2][1].split(":")[0]
     @origin   = host
-    @port     = request_lines.last[1].split(':')[1]
+    @port     = request_lines[2][1].split(":")[1]
     @accept   = request_lines[0][1]
+    unless request_lines.last[0].split("=").last.to_i == 0
+      @guess = request_lines.last[0].split("=").last.to_i
+    end
     self
   end
+
+  def build_output(output_body)
+    "<html><head></head><body>#{output_body}</body></html>"
+  end
+
+  def build_headers(output_length)
+    headers = ["HTTP/1.1 #{status_code}",
+               "date: #{Time.now.strftime('%a, %e %b %Y %H:%M:%S %z')}",
```

Comment:
Do not use `Time.now.strftime` without zone. Use one of `Time.zone.now.strftime`, `Time.now.strftime.current`, `Time.now.strftime.in_time_zone`, `Time.now.strftime.utc`, `Time.now.strftime.getlocal`, `Time.now.strftime.iso8601`, `Time.now.strftime.jisx0301`, `Time.now.strftime.rfc3339`, `Time.now.strftime.to_i`, `Time.now.strftime.to_f` instead.